classification/scripts/models.py

```diff
@@ -0,0 +1,191 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torchvision import models
+
+
+# --------------------
+# Utility Modules
+# --------------------
+
+class InitialConv(nn.Module):
+    def __init__(self, in_channels, out_channels):
+        super().__init__()
+        self.conv = nn.Conv2d(in_channels, out_channels, kernel_size=7,
+                              stride=2, padding=3)
+        self.bn = nn.BatchNorm2d(out_channels)
+        self.relu = nn.ReLU(inplace=True)
+
+    def forward(self, x):
+        return self.relu(self.bn(self.conv(x)))
+
+
+class MaskAttention(nn.Module):
```

Comment:
If this code is yours I'd rename this module since it's not attention, it's just masking.